src/library/scala/collection/immutable/Seq.scala

```diff
@@ -155,5 +154,4 @@ trait LinearSeqOps[+A, +CC[X] <: LinearSeq[X], +C <: LinearSeq[A] with LinearSeq
     with collection.LinearSeqOps[A, CC, C]
 
 /** Explicit instantiation of the `Seq` trait to reduce class file size in subclasses. */
-@SerialVersionUID(3L)
```

Comment:
I remember this one!

Reply:
I vaguely remembered the back and forth on serializing, but I was scared to see if I'm to blame.